docker/bitcoin-regtest/blockchaincfg.json

```diff
@@ -0,0 +1,21 @@
+{
+    "coin_name": "Regtest",
+    "coin_shortcut": "REGTEST",
+    "coin_label": "Regtest",
+    "rpc_url": "http://127.0.0.1:18021",
+    "rpc_user": "rpc",
+    "rpc_pass": "rpc",
+    "rpc_timeout": 25,
+    "parse": true,
+    "message_queue_binding": "tcp://127.0.0.1:48321",
+    "subversion": "",
+    "address_format": "",
+    "xpub_magic": 70617039,
+    "xpub_magic_segwit_p2sh": 71979618,
+    "xpub_magic_segwit_native": 73342198,
+    "slip44": 1,
+
```

Comment:
Unnecessary newline. But don't worry about it if it'll be the only change, I'll just fix it later.

Reply:
50eb2c6